src/methods/SANA.cpp

```diff
@@ -2376,176 +2376,217 @@ int SANA::MS3IncChangeOp(Job &job, uint source, uint oldTarget, uint newTarget)
 // Return the change in NUMERATOR of MS3
 int SANA::MS3IncChangeOp(uint source, uint oldTarget, uint newTarget) {
     if (MultiS3::denominator_type==MultiS3::rt_i){
-       /*int diff;
-        uint neighbor;
-        const uint n = G1AdjLists[source].size();
-        for (uint i = 0; i < n; ++i) {
-            neighbor = G1AdjLists[source][i];
-            diff = G2Matrix[oldTarget][(*A)[neighbor]] + 1;
-            if (MultiS3::denominator_type == MultiS3::rt_i){
-                MultiS3::denom -= diff;
+           /*int diff;
```

Comment:
If this code should stay, put it inside #if 0... #endif. Don't comment out long sections of code. If it's no longer needed, remove it. If it's supposed to be pseudo-code, replace it with a descriptive English comment.